packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/modal/ReactModalHostView.java

```diff
@@ -197,18 +198,18 @@ protected void setTransparent(boolean transparent) {
   }
 
   protected void setStatusBarTranslucent(boolean statusBarTranslucent) {
+    mPropertyRequiresNewDialog |= statusBarTranslucent != mStatusBarTranslucent;
```

Comment:
Can we avoid bitwise operations here? Otherwise LGTM.

Reply:
Sure thing!